docs/guides/building/_tauri-build.md

```diff
@@ -0,0 +1,8 @@
+
+import Command from '@theme/Command'
+
+To build and bundle your Tauri application into a single executable simply run the following command:
+
+<Command name="build" />
+
+It will build your Frontend, compile the Rust binary, collect all external binaries and resources and finally produce neat platform-specific bundles and installers.
```

Comment:
Do we need to mention that it will only build your frontend if it's configured to do so (beforeBuildCommand) ?

Reply:
Absolutely, we get some questions on that set of commands a lot so I think the more we link out to more information the better.